php-checks/src/test/resources/checks/UnusedFunctionParametersCheck.php

```diff
@@ -303,3 +303,27 @@ public function normalMethod($subject, $arg1, $arg2) // Noncompliant 3
         return;
     }
 }
+
+// Magic methods - methods starting with __ should not raise issues
+class MagicMethodsClass
+{
+    // Destructor - should not raise issues
+    public function __destruct()
+    {
+        return;
+    }
+
+    // Regular method starting with __ (but not a standard magic method) - should not raise issues
```

Comment:
```suggestion
    // FN: Regular method starting with __ (but not a standard magic method) - should not raise issues
```
Do I understand correctly that this is actually a false negative?

I think it's fine, because the doc clearly says
> All methods names starting with __ are reserved by PHP. Therefore, it is not recommended to use such method names unless overriding PHP's behavior.

and I hope it's not common to declare non-standard magic methods. But anyway, I think it's worth mentioning explicitly.